bpf/cgroup/bpf_cgroup_events.h

```diff
@@ -51,7 +51,7 @@ send_cgrp_event(struct bpf_raw_tracepoint_args *ctx,
 	memcpy(&msg->cgrp_data.name, &cgrp_track->name, KN_NAME_LENGTH);
 	probe_read_str(&msg->path, PATH_MAP_SIZE - 1, path);
 
-	perf_event_output(ctx, &tcpmon_map, BPF_F_CURRENT_CPU, msg, size);
+	PERF_METRICS(METRIC_CGROUP, perf_event_output(ctx, &tcpmon_map, BPF_F_CURRENT_CPU, msg, size));
```

Comment:
Is there a reason to use a macro here? can't we just use a (inlined) function?

E.g.,
```
perf_event_output_metric(ctx, METRIC_CGROUP, &tcpmmon_map, ...)
```

Reply:
Fixed